### ISSUE TITLE
Enhancement request: Next Stage Start time with offset days for different Regions

### ISSUE THREAD

**Issue by rasselmaliyekkal** [helderpinto/AzureUpdateManagerTools#5]
In our environment, we are having Dev machines only in central us, but production machines are there in many azure regions, so maintenance window is different.

Can we have a additional parameter in nextstageproperty for desired timezone and start time in hh:mm, so that it can create maintenance configurations in multiple regions?

**Comment by helderpinto** [comment]
Thanks for the suggestion, @rasselmaliyekkal. Would replacing the `offsetDays` parameter by a new `offsetTimeSpan` parameter, where one can specify the exact ISO 8601 offset in days/hours (e.g., `P7DT4H` for 7 days + 4 hours), address your need?

**Comment by rasselmaliyekkal** [comment]
Yes that way works I think, so i can create first maintenance configuration in UTC and calculate off set hours according to maintenance window of other regions.
I hope that will work, Thankyou ☺️

**Comment by helderpinto** [comment]
Thanks for the feedback, @rasselmaliyekkal. This enhancement will be implemented soon. This issue will be closed automatically once the code is merged into the main branch.
